src/components/header/Header.js

```diff
@@ -30,7 +33,7 @@ function Header() {
           </li>
           {exp === true && (
             <li>
-              <a href="#experience">Work Experiences</a>
+              <a href="#opensource">Open Source</a>
```

Comment:
Revert to `Work Experience`  and please make this check for all the components that only those with true show's on the header

Reply:
ready